libraries/joomla/form/form.php

```
@@ -1835,7 +1829,7 @@ protected function validateField($element, $group = null, $value = null, $input
 		// Make sure there is a valid JXMLElement.
 		if (!$element instanceof JXMLElement)
 		{
-			return new JException(JText::_('JLIB_FORM_ERROR_VALIDATE_FIELD'), -1, E_ERROR);
+			return new JException('Form field validation error', -1, E_ERROR);
```

I think we can throw an InvalidArgumentException here.